rules/getting-a-busy-person-into-the-meeting/rule.md

```diff
@@ -5,24 +5,38 @@ uri: getting-a-busy-person-into-the-meeting
 authors:
   - title: Chris Schultz
     url: https://ssw.com.au/people/chris-schultz
+  - title: Levi Jackson
+    url: https://www.ssw.com.au/people/levi-jackson
 created: 2021-12-16T23:49:23.696Z
 guid: 2c6e0175-0723-4f3b-81c8-33260f625a46
-
 ---
-
 It can be difficult to get everybody into a meeting, especially these days when many meetings are online. It is important that time is not wasted in these situations, so it's good to be proactive and have a plan for when the decision maker is busy.
 
 <!--endintro-->
 
 ### Make it a ceremony
 
 You have already [shared the agenda](/share-the-agenda), so pick out the best item and ping the decision maker on teams before the meeting: 
+
 > "We're meeting in 30 minutes, I'm excited to be talking about xxx"
 
 ### Be persistent
 
-If the meeting should have started and the decision maker has not joined, then add them to the call, ping them again, or call them. If you don't get a response, tell them *"I will call you towards the end of the meeting for a summary"*, as per [Do you know how to loop someone in at the end of a meeting?](/loop-someone-in)
+If the meeting should have started and the decision maker has not joined, then add them to the call, ping them again, or call them. 
```

Comment:
I think it needs to be clearer that we want them to add the people to the call, and only if they don't answer, then ping them.